src/main/java/it/gov/pagopa/pu/workflow/wf/debtposition/custom/activity/CancelReductionExpirationScheduleActivity.java

```diff
@@ -0,0 +1,11 @@
+package it.gov.pagopa.pu.workflow.wf.debtposition.custom.activity;
+
+import io.temporal.activity.ActivityInterface;
+import io.temporal.activity.ActivityMethod;
+
+@ActivityInterface
+public interface CancelReductionExpirationScheduleActivity {
+
+  @ActivityMethod
+  void cancelScheduling(String workflowId);
```

Comment:
let's always give meaningful name to activity method
```suggestion
  void cancelReductionPeriodExpirationScheduling(String workflowId);
```